Store menu (#1273)
aziontech/docs: src/content/docs/en/pages/store-journey/overview.mdx

---
title: Store Overview

> description: Explore Azion's Edge SQL and Storage for scalable data management.

Reviewer: ```suggestion
description: Explore Azion Edge SQL and Storage for scalable data management.
```

meta_tags: 'Edge SQL, Edge Storage, Azion, distributed database, data management, SQL queries, database resilience, scalable database, data processing, S3-compatible storage'
namespace: docs_edge_sql_journey
permalink: /documentation/products/store/overview/
menu_namespace: storeMenu
---

> Welcome to Azion's Store products journey documentation. Here, we introduce you to [Edge SQL]() and [Edge Storage](), our powerful tools for efficient data management and processing.

Reviewer: ```suggestion
Welcome to Azion Store Products journey. This documentation introduces [Edge SQL]() and [Edge Storage](https://github.com/aziontech/docs/pull/1273/files), powerful tools for efficient data management and processing.
```

> Welcome to the Azion Store Products journey. This documentation introduces [Edge SQL]() and [Edge Storage](https://github.com/aziontech/docs/pull/1273/files), powerful tools for efficient data management and processing.

Reviewer 2: ```suggestion
Welcome to the Azion Store Products journey. This documentation introduces [Edge SQL](/en/documentation/products/store/edge-sql/) and [Edge Storage]/en/documentation/products/store/edge-storage/), powerful tools for efficient data management and processing.
```

> Welcome to the Azion Store Products journey. This documentation introduces [Edge SQL]() and [Edge Storage](https://github.com/aziontech/docs/pull/1273/files), powerful tools for efficient data management and processing.

Reviewer 2: ```suggestion
Welcome to the Azion Store Products journey. This documentation introduces [Edge SQL](/en/documentation/products/store/edge-sql/) and [Edge Storage](/en/documentation/products/store/edge-storage/), powerful tools for efficient data management and processing.
```

## Edge Storage

> Azion Edge Storage is an object storage model that organizes data into "buckets" for efficient organization. It supports the S3-compatible API, classifies data operations into three classes, and incorporates data replication in its architecture. Key features include S3-Compatible API, Public Buckets, Integration with Edge Functions, Globally Distributed, Integration with Azion CDN, and Granular Permissions.

Reviewer: ```suggestion
Azion Edge Storage is an object storage model that organizes data into "buckets" for efficient organization. It supports the S3-compatible API, classifies data operations into classes, and incorporates data replication in its architecture. Key features include S3-compatible API, public buckets, integration with Edge Functions, globally distributed, Integration with Azion's network, and granular permissions.
```

### Edge Storage Use Cases & Applications

> Edge Storage is ideal for Backup and Recovery, Content Management, Rich Media, Static Site Hosting, Storage of Unstructured Data, and Jamstack Websites.

Reviewer: ```suggestion
Edge Storage is ideal for backup and recovery, content management, rich media, static-site hosting, storage of unstructured data, and Jamstack websites.
```
